notebooks/measures/Search_Download_and_Plot_GeoTIFFs.ipynb

```diff
@@ -0,0 +1,503 @@
+    "3. A basic understanding of python may also be helpful for this tutorial, but not necessary."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "25584ec8",
+   "metadata": {},
+   "source": [
+    "### **Example of end product**\n",
+    "\n",
+    "At the end of this tutorial, you will have produced a figure similar to the one below, which overlays velocity data on top of a digital elevation model:\n",
+    "\n",
+    "<center>\n",
+    "    <img src='./img/example_geotiff_plot.png'/>\n",
+    "</center>\n",
+    "    "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "30fbaa69",
+   "metadata": {},
+   "source": [
+    "### **Time requirement**\n",
+    "\n",
+    "This tutorial will take approximately 30 minutes to complete. \n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "90d526d0",
+   "metadata": {},
+   "source": [
+    "## **2. Tutorial Steps**\n",
+    "\n",
+    "### **Import libraries and classes**\n",
+    "\n",
+    "We will use the following libraries:\n",
+    "1. `earthaccess` to authenticate, search and download NSIDC DAAC data \n",
+    "2. `os` to list all the files we have downloaded \n",
+    "3. `osgeo.gdal` and `osgeo.osr` to crop and resample one of the GeoTIFFs \n",
+    "4. `rasterio` and `numpy` to read the GeoTIFFs and set up a grid for plotting the data. \n",
+    "5. `matplotlib` for plotting the data. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "da4e53c4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import earthaccess\n",
+    "import os\n",
+    "from osgeo import gdal, osr\n",
+    "import rasterio\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "from affine import Affine"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a94373e3",
+   "metadata": {},
+   "source": [
+    "### **Authentication**\n",
+    "\n",
+    "We need to set up our authentication using our Earthdata Login credentials and the `login` method from `earthaccess`. If we have a .netrc with our Earthdata login credentials, we can just run the cell below and it will automatically authenticate. If we don't have a .netrc, we will be prompted to enter our Earthdata Login username and password. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "257a1e51",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "auth = earthaccess.login()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6aad881a",
+   "metadata": {},
+   "source": [
+    "### **Search for data using spatial and temporal filters**\n",
+    "This tutorial assumes we already know which data sets we would like to download (that have data in GeoTIFF format). Each data set at NSIDC has a data set ID, it can be found after the title on the data set landing page. \n",
```

Comment:
```suggestion
    "This tutorial assumes we already know which data sets we would like to download (that have data in GeoTIFF format). Each data set at NSIDC has a data set ID; it can be found after the title on the data set landing page. \n",
```